cmd/containerize/containerize.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"teleport-exec/cgroups"
+	"teleport-exec/containerize"
+	"time"
+)
+
+func main() {
+	controller := containerize.NewController()
+	defer controller.Close()
+
+	if len(os.Args) < 2 {
+		fmt.Println("Usage: containerize.go command args...")
+		os.Exit(1)
+	}
+
+	// Prepare cgroups support
+	cgroups.Setup()
+	defer cgroups.TearDown()
```

Comment:
`TearDown()` will not be called in case of error. This is because you're using `log.Fatalln` everywhere, which calls `os.Exit`, skipping your deferred calls.

Reply:
Removed all Fatal calls